### ISSUE TITLE
Feature request, add "In Trakt Lists"

### ISSUE THREAD

**Issue by hcgiub001** [henryjfry/repository.thenewdiamond#27]
TheMovieDb helper has for each movie "In Trakt Lists"
Probably one of the best ways to find new content.
btw I was the one who asked you to include the IMDB recommendations like 3 years ago lol

the In Trakt Lists is just from the bottom of the page for movies.
Would you consider adding this somewhere in your Diamond Info?

thanks for your consideration

**Comment by henryjfry** [comment]
Show me a screenshot of what you mean?

I think you are saying you browse a movie and open the info screen and it shows the trakt lists that the current movie is on? And you then want to be able to go to that list?

If so then this would need a separate list like the current recommendations that would contain the Trakt lists? Or maybe a button which opens up a selection dialog with a list of lists?

If I were to add this is would be a button which opens up the "in trakt lists" list and you could then click on the item and load that list?

Is that what you are describing?
I dont particually want to add another GUI element, mostly because the GUI isnt mine and i know very little about kodi skinning. So i dont have too much appetite to try and create a whole new GUI section.

But another context menu option like "Similar" but for the lists could work.
And having it as a button would prevent another API call from having to be run by default

**Comment by hcgiub001** [comment]
Yes pretty much what you said.

below is loading Diamond info, then going on TMDBHelper and then inside TMDBHelper yoou have the In Trakt Lists
![Screenshot (208)](https://github.com/user-attachments/assets/7c625009-6524-4759-8bdb-1dda0ead752d)
![Screenshot (209)](https://github.com/user-attachments/assets/9f694325-36d5-4bab-b88c-786c0bf08df5)
![Screenshot (210)](https://github.com/user-attachments/assets/9d72d618-3f8f-4dc9-b237-6c911d54622b)

I was thinking you can add a button that loads into the list inside diamon info screen in the same row as play TMDBHelper Trakt
and also in context menu to load the list would be pretty cool.

These lists are pretty awesome. and good way to find similar content.

**Comment by henryjfry** [comment]
Shouldnt be too difficult, its getting lists of lists already from your trakt likes so adapting that to do one of the api calls:

https://api.trakt.tv/shows/%s/lists/all/popular
https://api.trakt.tv/shows/%s/lists/all/popular

And then load the selected list shouldnt be too difficult.

Ill look into it.

**Comment by hcgiub001** [comment]
do you which lists I'm talking about ? 
so for this movie here
https://trakt.tv/movies/furiosa-a-mad-max-saga-2024

the "In lists" are at the bottom of that movie
![Screenshot (211)](https://github.com/user-attachments/assets/80662971-0b80-4979-8780-850c0a9e36c6)

each movie has that at the bottom, it should be even easier than the way you are doing it?

**Comment by henryjfry** [comment]
Ok so ive got this basically working I think.

You can select "In Trakt Lists" from the context menu on main list screen for movies and tv shows and it returns like the first 10 or so more popular lists containing the item in a dialog box.
In the movie info screen you can select it from the collections or similar items context menu.
And in the TV show info screen you can select it from the "Play" context menu or the similar items context menu.

I've even got it integrated with the window stack and if you browse to a list you can return to the previous screen.

So you select the list and itll then reload the items on the list in the main list screen, just like when you load a regular trakt list.

I need to test this a bit but itll be added into the next version when i push that to the repo.

**Comment by hcgiub001** [comment]
awesome! just to let you know, I asked Fenlight to look into adding IMDB recommendations, which he added to his Extras Menu.
I also asked him to look at adding the trakt lists into his Extra's Menu, which he did
here is an example
https://streamable.com/1obrhe
I took out the animations so it's faster. 
The reason why I asked you also was because your information menu is universal, whereas Fenlight is only for Fenlight. 
anyway thought you could compare and get ideas of others

I'm curious on this "I've even got it integrated with the window stack and if you browse to a list you can return to the previous screen."

I wanted to return to extras menu in Fenlight from going into a list, so that would be cool.
By the way. I was worried to ask Fenlight Dev to add imdb because of load times, but he has everything running in parallel. it has zero delay on the lists he was already populating.
I know your addon if you chose to include IMDB list it takes a significant time longer.
I was wondering if you knew how to multithread like Fenlight does. the speed differential is like 0.2 seconds compared to 4 seconds.
his addon is here if you haven't already got it
https://github.com/Tikipeter/tikipeter.github.io

if you can have it instant like Fenlight Extra menu then this is a game changer. I'll make a video about your addon after you updated to let others know about it. peace

**Comment by henryjfry** [comment]
Yeah id have to rebuild everything to make it work like that probably. And there is an awful lot of skin stuff I just dont know enough about.
If you have any ideas how fenlight is faster than mine by all means compare the code and tell me how fenlight does it.

But im not actually a programmer by training.  There is a tonne of core concepts I just dont know and ive picked this stuff up as ive gone along. 

It works as is, thats basically all i care about.

**Comment by hcgiub001** [comment]
LOL I won't be able to set up a github account. The Dev of Fenlight Tikipeter has really started to push the addon. I don't think it's Fenlight anymore and more like Fen++ (I'm going to ask him to change to this LOL)
the multi episode playback I don't think Fenlight does. but I can ask him in the future.
I think you are an amazing Dev , seen your posts of the stuff you've done!
I'm going to make a post about Dimond_info addon. lots of people could make use of it. looking forward to the update . peace

**Comment by henryjfry** [comment]
When I said multi episode i meant continuous playback. Itll flash up a fenlight screen between episodes. And the popup next episode if you stop or cancel it just plays the next episode.

I dont like that

**Comment by hcgiub001** [comment]
it has so many cool features that I think this should be included. thought about asking Tikipeter himself? I've got like 3 feature requests right now so I don't think I should ask lol (I also got like 4 others that I'm waiting to slowly ask)
I think slowly he could keep adding things to make it better.
I do understand what you are saying. that would annoy me to. which addon are you using? Seren?

**Comment by henryjfry** [comment]
Ive looked at fenlight and the speed differences are not due to the IMDB side but the TMDB side.
Getting the information from IMDB is equally quick on fenlight vs mine, its the same URL and it needs parsed as an html page. But once the list of IMDB's has been retrieved there is then no way to retrieve this information from the cache in extended info or from TMDB but as individual requests.
The cache is simple text files as thats how it was always setup and I suspect the difference is there.

Mine are all individual lookups as they were cached, presumably fenlight is too but itll be running on an open database which is considerable quicker.

**Comment by henryjfry** [comment]
I could potentially rebuild the caching mechanism to store everything in SQL db. Probably wouldnt be very difficult but there would then be file lock stuff to consider.
I do a lot of SQL queries for my job but not so much actual DB writes so it would need to be able to read/write to the DB and keep the connection/reopen as needed.

Opening and closing the db on demand probably wouldnt be too difficult, already do this for trakt lookups which were very slow without a DB to operate on.  But to keep it open during any load steps would be the kicker.

I might look into it eventually.

**Comment by hcgiub001** [comment]
> I could potentially rebuild the caching mechanism to store everything in SQL db. Probably wouldnt be very difficult but there would then be file lock stuff to consider. I do a lot of SQL queries for my job but not so much actual DB writes so it would need to be able to read/write to the DB and keep the connection/reopen as needed.
> 
> Opening and closing the db on demand probably wouldnt be too difficult, already do this for trakt lookups which were very slow without a DB to operate on. But to keep it open during any load steps would be the kicker.
> 
> I might look into it eventually.

you're addon is fast, it's only when you enable IMDB recommendations. Tikipeter mentioned "Even so, those things can be threaded for speed, and since each list within Extras is also threaded, it doesn't slow the display of the Extras window at all."

in Fenlight all lists are threaded so the loading of the list is probably as fast as diamond.info but because it's threaded, it loads up slightly later in the menu and I still have other lists I can explore while it loads.

**Comment by henryjfry** [comment]
Well extended info does do threading when its loading but mostly that has nothing to do with me and whatever it was doing before I got to it.
But im pretty sure when you click on a new screen there should be a thread for each element on the screen (including the "similar" section) and the elements to load to it arent added until the thread is complete?

I dont really understand threading in python, I have used it and added a couple of specific threaded functions in the service module, as im aware that kodi processes stuff in sequence and a long plugin call can hold stuff up.

But thread locks, stopping, starting, restarting and killing threads are aspects of threads i understand conceptually but in python dont seem to be very user friendly and i have troube implementing.


I have been noodling around setting up a SQL cache which would effectively contain all the current TXT cache entries but only in SQL.  And should *theoretically* be kept open while the addon is open and provide efficencies in that regard.
But its literally a wrapper and container for the current TXT cache which was included in extendedinfo I took it over.

So no gained effeciency from the stored cache entries being parsed.  Its doing exactly as many cache lookups and loops through stuff.

So if an imdb watch list has been populated and not expired, that lookup is fairly quick.  But if you look at a tv show with 12 seasons it has to loop every trakt API call and check which episodes are unwatched. If thats cached its quicker but it still has to do it every time, although now I think of it I could actually cache that actvity.

Anyway long story short, i can store what it already does in a db which might speed things up a bit.  But a tonne of stuff is pretty ineffecient, without any help im not inclined to rewrite a lot of stuff to save the seconds here and there.
Actual records written to a db would speed stuff a lot probably but it would require a hell of a lot of testing if not actual help.  And i get neither.


A bunch of speed has

**Comment by hcgiub001** [comment]
https://github.com/user-attachments/assets/816e3b33-e1ef-4027-b534-c375daf6088d

you can see that the IMDB recommendations actually take the longest to load so they come later but because everything is threaded it doesn't matter on the slowest element. Also I modified the extras Menu to remove the animation, that itself would help diamond.info, the animation I'm not fond of

when I asked Fenlight Dev, he wrote
"It is slower than TMDb recommendations, because of the fact IMDb (webpage) is a lot slower to respond than TMDb's API, and also because I would need a call to TMDB for each of the 13 results to get details enough to use. The only thing returned from IMDB is the 13 IMDb id's e.g. ['tt908974', 'tt39847362'......etc]. In the case of movies, that results in 13 calls to TMDB to get each item's details. In the case of TV Shows, it results in 26 calls as there is no mechanism to immediately get TV Show data using an IMDB id. With movies there is.
Even so, those things can be threaded for speed, and since each list within Extras is also threaded, it doesn't slow the display of the Extras window at all."

Your addon was fast without IMDB recommendations so you must not thread and you are waiting for the IMDB to finish and then you take another little while for animation.

I'm not a programmer or coder, I only see that you are waiting for IMDB list to populate?
plus you sort the order so you need all the list before you can perform any logic on it?
I feel as though Fenlight knows their is a maximum or 12 items and it threads each item so they won't all come at the same time and we just have them as the order that is set in IMDB? 
the issue is if you can load diamond.info the same as with and without IMDB recommendations, because without them it's fast

**Comment by henryjfry** [comment]
Ok so the IMDB stuff wasnt in a thread, ive added that now and I've also changed over the cach to a SQL database.

So im testing that but it does seem to be faster

**Comment by hcgiub001** [comment]
that should really just be it I think, this could be great now. I will recommend it because before it was kind of slow. 
I really dislike the animation, every time we click on an element we have to go out of the diamon.info panel and then back in. It's ugly imo, can you just have an instant load to other screen?
also would be insane if when we can long click on an item to play in dimond.info , we can have an dialogue asking what player you want to select.

**Comment by hcgiub001** [comment]
are each of the items within a list also threaded like Fenlight? then you also thread the lists too , right?

**Comment by henryjfry** [comment]
> that should really just be it I think, this could be great now. I will recommend it because before it was kind of slow. I really dislike the animation, every time we click on an element we have to go out of the diamon.info panel and then back in. It's ugly imo, can you just have an instant load to other screen? also would be insane if when we can long click on an item to play in dimond.info , we can have an dialogue asking what player you want to select.

Ok so there will **always** be a closeing and reopening of the extendedinfo screen.  Thats a specific change I made because the window stack used to make video playback jerky once you opened more than a couple of windows and the only way to ensure kodi actually released the resources was to destroy the window every time. 
So no i shant be undoing that im afraid. I dont run kodi on a pc, perhaps that was never an issue on a pc? But it was always been an issue with me on a pi or other linux based hardware.
And its partly why i started playing with Openinfo in the first place.

Long click? Thats a kodi bulitin thing I think, firstly you need a remote that can actually do a long click, I think my OSMC remote can, but an IR remote definetly cant and you just map a function to it in keymap. Other than that how would I even tell that a long click event is happening? 
Does Fen do that or are you just fantasising now?

Habe you looked in '/script.extendedinfo/resources/lib/DialogVideoList.py'
I mapped a bunch of pageup/pagedown to all the buttons on the left side of the list screen so you can actually pgdn/pgup on that list like an actual list (which you cant do on a panel in kodi apparently without that kind of hack)

So I could theoretically map a function to like 'context play' or something but i use my kodi box mostly for watching TV and i do that mostly from the "Trakt Watched TV" list, so when you click 'play' or otherwise launch the "context play" function what does that mean?
Next episode, next episode rewatch, first episode?
Theres a reason why i added those 3 options and your longclick play would only work for movies. Could be a setting I suppose for TV, i might look into building the function similar to the context_info or whatever I built.  But the longclick would be a keymap. Look at the OSMC keymap for guidance:

https://github.com/xbmc/xbmc/blob/master/system/keymaps/osmc/osmc_remote.xml

> are each of the items within a list also threaded like Fenlight? then you also thread the lists too , right?

What? I dont know, not sure what you are talking about. But i just added the imdb stuff to a thread, whatever it did before it still does now.
My cache calls will specifically not be threaded beyond that because ive already had issues with accessing the SQL from more than 1 thread.

So im going to need you to test this.  Its crashed on me because of the changes and I cant just push that out without actually testing it.

Also i dont really want you "promoting" it.  Sure post on reddit how cool my addon is but i dont know if people watch your videos but if you do i dont need a bunch more people looking for support.

Unless you are going to actively provide support as a longer term user of the addon.

**Comment by henryjfry** [comment]
[script.extendedinfo-8.38.zip](https://github.com/user-attachments/files/16624662/script.extendedinfo-8.38.zip)

**Comment by hcgiub001** [comment]
https://streamable.com/iwgjjc

![Screenshot (357)](https://github.com/user-attachments/assets/8e13b03c-81cd-4270-b54c-7107480eff6e)

in the review I can't see anywhere where it takes me to trakt lists. the trakt lists I opened at the end is from Fenlight.

> In the movie info screen you can select it from the collections or similar items context menu.

I thought of checking context menu in Umbrella addon since you said

> You can select "In Trakt Lists" from the context menu on main list screen for movies

and Fenlight used same wording too.

also in the video you will see I've only used TMDB recommendations and it's a second after the script has started, then I did only TMDB and it's more like 4 seconds, so to me their is no difference, it's waiting on IMDB results, Fenlight isn't waiting on anything, it loads the Extra Menu same time if I have IMDB recommendations on or not.

I looked into settings section to see if I might not have had something enabled, but you can see I have everything enabled?

**Comment by henryjfry** [comment]
Its a context menu option. So on the video list you click the context menu item "in Trakt Lists" and it brings up a list of the lists which you can then open.


So i added it to the videoinfo "play" button and the similar items context menu button. For TV shows its not accessible from the info screen yet I dont think

**Comment by hcgiub001** [comment]
> Does Fen do that or are you just fantasising now?

https://github.com/user-attachments/assets/e522438f-1ba1-483f-8d4b-c56ac6521c22

It is possible to have an action when  long press select. When you hover over a trakt list in Extra Menu in Fenlight , it will perform the action "like list"

not really sure where you can use this but it's their if you wanted to explore

> What? I dont know, not sure what you are talking about. But i just added the imdb stuff to a thread, whatever it did before it still does now.

Fenlight Dev wrote 

> I would need a call to TMDB for each of the 13 results to get details enough to use.,.., 26 calls as there is no mechanism to 
> immediately` get TV Show data using an IMDB id, Even so, those things can be threaded for speed, and since each list within 
> Extras is also threaded, it doesn't slow the display of the Extras window at all.

so if I'm reading correctly each movie has has a call, (twice for shows). each call is threaded so it doesn't slow it down.
then each list (TMDB reccommendations, IMDB reccommendations, Cast) are themselves threaded.

I was asking you if you did similar. 

I've mentioned Fenlight doesn't wait for IMDB reccommendations, since diamond.info is taking longer to load with IMDB, then it's waiting for the results?

> Sure post on reddit how cool my addon is but i dont know if people watch your videos but if they do i dont need a bunch 
> more people looking for support.

I just make tutorial videos on reddit on things that I like about Kodi.

here is a post I did that I thought of quickly making something for diamon.info
https://www.reddit.com/r/Addons4Kodi/comments/1erfnpp/fenlight_nimbus_combined_allows_you_to_see_and/

it's nothing big

**Comment by hcgiub001** [comment]
> Its a context menu option. So on the video list you click the context menu item "in Trakt Lists" and it brings up a list of the lists 
> which you can then open.

I have no context menu option that says "In Trakt Lists"
I made a video of what I see
https://streamable.com/1yu7fi

**Comment by henryjfry** [comment]
Is that a skin?
I added functions within my addon, not global options.

Open extendedinfo on a list, open the extendedinfo context menu, "in trakt lists"

Thats what you asked for? If you want me to add stuff to your skin? No.
I probably could add a global context option but jesus christ did you make it unclear thats what you were actually talking about.

and you are kind of becomming insufferable now.

**Comment by henryjfry** [comment]
And have you installed [script.extendedinfo-8.38.zip](https://github.com/user-attachments/files/16624662/script.extendedinfo-8.38.zip) for testing

Im not going to push any more updates to my repor until its well tested, **by you**.

**Comment by henryjfry** [comment]
See id rather add context menu options than extra buttons because you need to add the button in all the xml files and its extra work

Id rather add extra functions as context menu items

Again you dont like it, learn how to make the changes youself and submit a PR

**Comment by henryjfry** [comment]
So whats the performance increase looking like?

**Comment by hcgiub001** [comment]
sorry been testing none stop , I've been messaging you and testing this since I commented 2 hours ago. literally not moved out of place

I don't use dimond.info from programmes. I use it on other addons (context menu, diamond info), so I was remembering how to use diamond.info

I also wanted to make sure the results align with what trakt gives, if they don't I check what Fenlight gives.
I tested shows and movies

I also wanted to test shows from a different language.
then I tested IMDB reccommendations

so my results , I didn't have any shutdowns, except one but It's probably my end because I have had a lot of things opened.

It works really well. I like that you chose a dialogue. I like that it goes remembers the window.
Some movies I checked the result of the lists and strangely it will have the first few the same as the what comes up in trakt (sorted by popularity) but then some options differ

like this show 
https://trakt.tv/shows/family-guy/lists

![Screenshot (361)](https://github.com/user-attachments/assets/cd63ad8e-61d2-4552-bb78-5a36bee0a558)

![Screenshot (362)](https://github.com/user-attachments/assets/8e959966-5d8f-4fd3-9389-a481c6aaa4c8)

this is actually only one of but I've had list where the bottom 5 or so are different but the titles in the list did have similar themes to the movie so I thought maybe it's still picking from the lists. Fenlight just has exact copies 

here is another movie that It has none of the top 10 from trakt lists
movie is called Black Noise
![Screenshot (365)](https://github.com/user-attachments/assets/53c58a19-87e3-4b00-bf28-cb2c32489bc5)
![Screenshot (364)](https://github.com/user-attachments/assets/80ae16d8-20f7-4af4-8997-332b65c12ed4)

again the themes are kind of matching so maybe you are not picking top 10 from popularity?

I noticed with lists that very big items , e.g. https://trakt.tv/movies/godzilla-x-kong-the-new-empire-2024 which has 3321 items
it takes allot of time. I think I waited 4+ minutes and then pressed backspace. In Fenlight he has a number next to list to show how many in list so maybe you can add that so we don't click on them because it's not really practical to wait 5 minutes for a list to load, or you can take those lists out.

I tried with shows and still good

I noticed a bug. If I use context menu and click on "In Trakt Lists" but don't pick one and press exit. it will bring up side menu, then I have to press back to close the side menu. it happens when I don't click on a list.

I tested with foreign shows and I don't think it brings them correct
so here
https://trakt.tv/shows/home-with-kids

in trakt it has only 1 "in trakt list" but diamond info brings up 10 and none of them are the one in trakt website.
one thing that is puzzling is that in trakt it says it should have 41 lists but when you click on it, I can only see one. so are you getting lists that we can't see?

Fenlight only shows 1 list which is the one on trakt

also weird thing is, it says kids show in lists that it brings up and that chinese show is a kids so it's not far off.
mostly for the foreign stuff I can't get actual matches with trakt

then I tested IMDB recommendations and as far as I can see, it's not much better, I can't tell at all. I then only did TMDB reccommendations and it's instant. then only did IMDB recommendations and it's like 3 seconds before it populates.

let me know if you need more testing. I think the feature is worth having, when it works. and no crashes but with big lists it's unusable, so we don't know how long it will take to populate, most lists are probably not that big (3000)

**Comment by hcgiub001** [comment]
Yeah that's fine that you rather keep it in your addon, I forgot how fun your addon was

> Is that a skin?

yes It's Nimbus skin, really nice skin that just had updates to make it look like that, it's not heavy either, I recommend to try it

**Comment by henryjfry** [comment]
Im picking whatever the results of:
````
    if type == 'movie':
        url = 'https://api.trakt.tv/movies/%s/lists/all/popular' % (imdb_id)
    else:
        url = 'https://api.trakt.tv/shows/%s/lists/all/popular' % (imdb_id)
````

Are, so lists containing the IMDB number of all types sorted by popularity.

Literally whatever trakt returns.

**Comment by hcgiub001** [comment]
> I noticed with lists that very big items , e.g. https://trakt.tv/movies/godzilla-x-kong-the-new-empire-2024 which has 3321 items
> it takes allot of time. I think I waited 4+ minutes and then pressed backspace

search for this movie godzilla-x-kong-the-new-empire-2024, and bring up context menu and click on "in trakt lists" and click on the top of the list called Latest releases.

this is what the in trakt lists in trakt is
![Screenshot 2024-08-15 151351](https://github.com/user-attachments/assets/c0a295fd-e763-4521-b1b6-d571817c889a)

I'm guessing it's that list which I thought had 3000 items but that list only has148 items (3000+ likes)

this is when I clicked on it to test again
![Screenshot (367)](https://github.com/user-attachments/assets/ee1ef0c9-46e0-4a40-8cc6-1467306dc4ab)

3:07
and now
![Screenshot (368)](https://github.com/user-attachments/assets/3963d3c0-244c-4cd1-902f-e80d4240802d)
I let it run for 10 minutes and the list is still not populating

**Comment by henryjfry** [comment]
Are you talking about returning a list of imdb recommendations where it hasnt been generated before?
Because 3 seconds is pretty good considering what it has to do.

TMDB items are returned as a block, but all the imdb listitems need to be created individually in a loop of however many.  Its now threaded so it starts doing that and it doesnt hold up the other parts of the screen but thats how it loads.

**Comment by hcgiub001** [comment]
yeah you're right, I was just comparing it to TMDB only recommendations, but can't be helped since we can't use API calls like we could with TMDB

**Comment by henryjfry** [comment]
I dont know what you are talking about re: speed.  Here is the same list I believe loaded in extended info and it loads very quickly:

https://streamable.com/t5hq77

Thats me starting from the trakt watched TV script url:

RunScript(script.extendedinfo,info=trakt_watched,trakt_type=movie)

And I tried a very long list which was already cached and it loads quickly.

On the fancy GUI screen it only ever loads 20 at a time.
If its loading like a plugin thats when it has to evaluate evey single item, I think unless you specify a limit in the plugin call you make.  I cant remember if I ever actually implemented that functionality as it would only ever be useful if you want a widget and other addons do that better.

**Comment by hcgiub001** [comment]
> search for this movie godzilla-x-kong-the-new-empire-2024, and bring up context menu and click on "in trakt lists" and click on 
> **the top of the list** called Latest releases.

you clicked 3rd in the list (which is also called latest releases)

I also don't get the numbers of movies in the list like you did
![Screenshot (369)](https://github.com/user-attachments/assets/92db056c-cfc8-4c6e-bdbd-042fed3aebaf)
on a side note, your addon loads searches very fast (movie searches)

**Comment by henryjfry** [comment]
The numbers are on the latest version

**Comment by henryjfry** [comment]
Current version script.diamondinfo v.8.42
Just fixed that issue

**Comment by henryjfry** [comment]
Also new icon

**Comment by henryjfry** [comment]
Added a general context menu item for movies and TV "Diamond Info - In Trakt Lists"

**Comment by henryjfry** [comment]
So this now exists im going to close it.

**Comment by hcgiub001** [comment]
sorry forgot to mention that Fenlight already has "In Trakt Lists" for their naming, Could you rename it possibly?
but thanks for pushing it through :)

**Comment by henryjfry** [comment]
That's why I called it "Diamond Info - In Trakt Lists"

Surely that differentiates it well enough

**Comment by hcgiub001** [comment]
perfect thanks :)
